pkg/wasm/imagefetcher.go

```diff
@@ -73,7 +74,7 @@ func NewImageFetcher(ctx context.Context, opt ImageFetcherOption) *ImageFetcher
 	}
 
 	if opt.Insecure {
-		t := remote.DefaultTransport.Clone()
+		t := remote.DefaultTransport.(*http.Transport).Clone()
```

Comment:
redundant cast

Reply:
It's not.. the changes the type to an interface that doesn't have Clone but the underlying type has Clone.  

Their own library uses the exact same code as here